src/app/zap-templates/zcl/zcl.json

```diff
@@ -305,7 +305,9 @@
             "PacketTxCount",
             "TxErrCount",
             "CollisionCount",
-            "OverrunCount"
+            "OverrunCount",
+            "FeatureMap",
```

Comment:
since we read FeatureMap from ZAP in clusterInit this canot be AAI only.

WE have to essentially waste a u32 of RAM to keep the default ember value for now.